_global/css/board_front/growth-options.css

```diff
@@ -389,12 +391,19 @@ growth-cell gain icon.sun,
 growth-cell gain icon.moon {
   background-repeat: no-repeat;
   background-size: contain;
-  display: block;
+  display: inline-block;
```

Comment:
I found this to be required to make the elements appear adjacent to one another instead of on top of one another. I'm not sure if this will later cause issues.

Also, on second glance, unsure why I only changed the .moon icon and why it worked regardless...